core/src/text/fontContext.cpp

```diff
@@ -185,6 +185,11 @@ bool FontContext::layoutText(TextStyle::Parameters& _params, const std::string&
                      (metrics.aabb.y + height * 0.5) * TextVertex::position_scale);
 
     auto it = _quads.begin() + quadsStart;
+    if (it == _quads.end()) {
```

Comment:
The _quads are set during textWrap/drawShapeRange. 

Actually the line.shapes().size() test at the beginning already checks for no-glyphs-at-all. So this new test only catches lines for which only whitespaces were found (and this really happend when no fallbacks were loaded, making the map unusable slow for Cairo). 

Should we rather drop labels if they have any missing glyphs?